src/main/java/com/epam/aidial/core/controller/AccessControlBaseController.java

```diff
@@ -45,4 +48,21 @@ public Future<?> handle(String bucket, String filePath) {
 
     protected abstract Future<?> handle(ResourceDescription resource);
 
+    protected boolean hasReadAccess(String bucket, String filePath) {
```

Comment:
this method only checks read access for shared files, and if I'm a file owner - read access is false, it might be confusing, we should either add a comment/javadoc or rename this method to `isSharedWithMe` or so

Reply:
fixed renamed to `isSharedWithMe `